website/blog/2020-08-04-prestodb-and-hudi.md

```diff
@@ -0,0 +1,153 @@
+---
+author: Bhavani Sudha Saktheeswaran
+authorURL: https://www.linkedin.com/in/bhasudha
+title: PrestoDB and Hudi
+---
+**co-author:** [Brandon Scheller](https://www.linkedin.com/in/brandon-scheller-a00851ab)
+
+[Apache Hudi](https://hudi.apache.org) is a fast growing data lake storage system that helps organizations build and manage petabyte-scale data lakes. Hudi brings stream style processing to batch-like big data by introducing primitives such as upserts, deletes and incremental queries. These features help surface faster, fresher data on a unified serving layer. Hudi tables can be stored on the Hadoop Distributed File System (HDFS) or cloud stores and integrates well with popular query engines such as [Presto](https://prestodb.io), [Apache Hive](https://hive.apache.org), [Apache Spark](https://spark.apache.org) and [Apache Impala](https://impala.apache.org). Given Hudi pioneered a new model that moved beyond just writing files to a more managed storage layer that interops with all major query engines, there were interesting learnings on how integration points evolved .
+
+In this blog we are going to discuss how the Presto-Hudi integration has evolved over time and also discuss upcoming file listing and query planning improvements to Presto-Hudi queries.
+
+<!--truncate-->
+
+
+## Hudi overview
+Apache Hudi (Hudi for short, here on) allows to store vast amounts of data, on top of existing DFS compatible storage, while providing two primitives, that enable stream processing on data-lakes, in addition to typical batch-processing.
```

Comment:
Apache Hudi (Hudi for short, here on) enables storing vast amounts of data on top of existing DFS compatible storage while also enabling stream processing in addition to typical batch-processing. This is made possible by providing two new primitives.